debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -205,7 +205,7 @@ public String knownGtidSet() {
     public GtidSet purgedGtidSet() {
         AtomicReference<String> gtidSetStr = new AtomicReference<String>();
         try {
-            jdbc.query("SHOW GLOBAL VARIABLES LIKE 'gtid_purged'", rs -> {
+            jdbc.query("SELECT @@global.gtid_purged", rs -> {
                 if (rs.next() && rs.getMetaData().getColumnCount() > 1) {
```

Comment:
When I tried on MySQL 5.7, `SELECT @@global.gtid_purged` returned only the value column unlike `SHOW` which returns key and value columns.

Reply:
@seruman Thanks for pointing out, I've added a test to assert the codepath